Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -62,6 +67,30 @@ private async void OnClosing(object sender, CancelEventArgs e)
             Environment.Exit(0);
         }
 
+        private void onPreviewCanExecute(object sender, CanExecuteRoutedEventArgs e)
+        {
+            // Allow paste command to accept non-text items
```

Comment:
what does this mean sorry, pasting non-text into the query window?

Reply:
Yeah, normally TextBox elements ignore non-text data. With this we intercept all data input and capture it.

Reply:
oh so this is just required so you can retrieve path from the file object in clipboard?